packages/styles/scss/components/treeview/_treeview.scss

```
@@ -74,19 +74,19 @@
       align-items: center;
 
       &:hover {
-        background-color: $hover-ui;
-        color: $text-01;
+        background-color: $background-hover;
```

Since the enabled background use `$layer-01` the hover should use `$layer-hover-01`

```suggestion
        background-color: $layer-hover-01;
```